src/ModelTraits/SpatieTranslatable/HasTranslations.php

```diff
@@ -65,10 +66,10 @@ public static function create(array $attributes = [])
             if ($model->isTranslatableAttribute($attribute)) { // the attribute is translatable
                 $model->setTranslation($attribute, $locale, $value);
             } else { // the attribute is NOT translatable
-                $model->{$attribute} = $value;
+                $non_translatable[$attribute] = $value;
             }
         }
-        $model->save();
+        $model->fill($non_translatable)->save();
```

Comment:
I see your point.

I got confused talking about update method. You did about create.

Anyway I don't agree to log a message if Entity is not saved. When you create a new Model (from Builder::create) or __construct() the fill method is called and if the model is FulledGuarded an Exception is thrown.

If it is not full guarded, the fillable attributes are filled and saved. The rest of attrributes passed to fill method are ignored. It would not be my first time I get crazy looking for the reason an attribute is not saved. The filled array is the reason.

So, IMHO, if Laravel behaves like this, would be nice that this Trait does not change this behaviour.

An alternative would be to call `new static ($attributes)` instead of `new static()` in line 62.

This Laravel constructor would throw the commented Exception if Model is fulled guarded or filled otherwise.

The thing is, the actual code of create and update applied in an application with users allows to create or update entities that do not belong to you or to update fields you are not allowed to. You can inspect the form and add/delete/change form fields. 

Do not you agree?